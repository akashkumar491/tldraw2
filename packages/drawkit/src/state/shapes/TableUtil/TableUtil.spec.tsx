import { Table } from '..'

describe('Table shape', () => {
  it('Creates a shape', () => {
    expect(Table.create({ id: 'table' })).toMatchSnapshot('table')
  })
})
