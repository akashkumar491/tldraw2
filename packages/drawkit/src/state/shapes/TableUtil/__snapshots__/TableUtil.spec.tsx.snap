// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table shape Creates a shape: table 1`] = `
Object {
  "childIndex": 1,
  "id": "table",
  "name": "Table",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "style": Object {
    "color": "black",
    "dash": "draw",
    "font": "script",
    "isFilled": false,
    "opacity": "medium",
    "scale": 1,
    "size": "small",
    "textAlign": "middle",
  },
  "text": " ",
  "type": "table",
}
`;
